clients/ops/admin-ui/src/features/auth/auth.slice.ts

```diff
@@ -56,27 +52,6 @@ export const selectToken = (state: RootState) => selectAuth(state).token;
 
 export const { login, logout } = authSlice.actions;
 
-export const credentialStorage = createListenerMiddleware();
```

Comment:
Removed the following event listeners given the redux-perist localStorage functionality. Redux-persist localStorage root key is `persist-root`.